.circleci/config.yml

```diff
@@ -254,8 +254,9 @@ commands:
       - run:
           name: Find PRs
           command: |
+            DAYS=3
             REPO_API_URL=$(echo "${CIRCLE_REPOSITORY_URL}/" | sed 's|.git/$||' | sed 's|git@github.com:|https://api.github.com/repos/|')
-            curl --fail --show-error "${REPO_API_URL}/pulls?state=open" | jq '.[].number' > open_prs.txt
+            curl --fail --show-error "${REPO_API_URL}/pulls?state=open" |  jq ".[] | select((.updated_at | fromdate) > (now - $DAYS*24*60*60)) | .number" > open_prs.txt
```

Comment:
Unfortunately this is a bit opaque - it uses `jq` 's filter abilities to only include PRs whose updated_at field is within the threshold (currently 3 days)